validphys2/src/validphys/process_options.py

```diff
@@ -359,10 +377,17 @@ def _dybosonpt_xq2map(kin_dict):
 DY_PT = _Process(
     "DY_PT",
     "DY W or Z (2 leptons) + j boson transverse momentum",
-    accepted_variables=(_Vars.pT, _Vars.m_W2, _Vars.m_Z2, _Vars.sqrts),
+    accepted_variables=(_Vars.pT, _Vars.m_W2, _Vars.m_Z2, _Vars.sqrts, _Vars.y, _Vars.m_ll2),
     xq2map_function=_dybosonpt_xq2map,
 )
 
+DY_PT_RAP = _Process(
+    "DY_PT",
+    "DY W or Z (2 leptons) + j boson transverse momentum",
+    accepted_variables=(_Vars.pT, _Vars.m_W2, _Vars.m_Z2, _Vars.sqrts, _Vars.y, _Vars.m_ll2),
```

Comment:
The function accepts also eta I believe.

Reply:
Ok, done!